### PR TITLE
Remove UserDataHandler form this project. Update pom

### PR DESCRIPTION
**Pull request by tchemit**
I can't use anylonger on my project the version 1.1.6 since it duplicates the UserDataHandler class with xml-apis project.

### CODE REVIEW

**Comment by elharo** [comment]
FYI, this makes perfect sense. In 2017 Java 1.4 support isn't important. I need to find a bit of time to think about licensing (we lost some legal bureaucracy when codehaus went down), merge this, and prepare a release.

File: pom.xml [hunk 1]
```diff
@@ -12,12 +12,13 @@
   <artifactId>jaxen</artifactId>
   <packaging>bundle</packaging>
   <name>jaxen</name>
-  <version>1.1.6</version>
+  <version>1.1.7-SNAPSHOT</version>
```
Comment (elharo) [inline pom.xml]: I'm thinking we should probably call the next release 1.2.0 since it changes the required Java version.

File: pom.xml [hunk 2]
```diff
@@ -12,12 +12,13 @@
...
   <description>Jaxen is a universal Java XPath engine.</description>
   <url>http://www.cafeconleche.org/jaxen</url>
   <licenses>
     <license>
-      <url>http://www.cafeconleche.org/jaxen/license.html</url>
+      <name>BSD-License</name>
+      <url>https://raw.githubusercontent.com/jaxen-xpath/jaxen/master/LICENSE.txt</url>
```
Comment (elharo) [inline pom.xml]: guess we lost that when the site moved. I'm not sure that's actually accurate. I'll have to check.

File: pom.xml [hunk 3]
```diff
@@ -338,25 +340,26 @@
     <dependency>
       <groupId>xml-apis</groupId>
       <artifactId>xml-apis</artifactId>
-      <version>1.3.02</version>
+      <!-- Do not use version 2.0.0 nor 2.0.2, it miss ElementTraversal class needed for example by maven help mojo -->
```
Comment (elharo) [inline pom.xml]: Are these dependency updates necessary too for 1.5?

File: pom.xml [hunk 4]
```diff
@@ -338,25 +340,26 @@
...
       <artifactId>xercesImpl</artifactId>
-      <version>2.6.2</version>
+      <version>2.11.0</version>
       <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>xom</groupId>
       <artifactId>xom</artifactId>
-      <version>1.0</version>
+      <version>1.1d2</version>
```
Comment (elharo) [inline pom.xml]: This one definitely shouldn't change.

File: pom.xml [hunk 5]
```diff
@@ -338,25 +340,26 @@
...
       <optional>true</optional>
     </dependency>
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>3.8.2</version>
+      <version>4.12</version>
```
Comment (elharo) [inline pom.xml]: We shouldn't change this here either.

**Comment by shweta21084** [comment]
Has the new jaxen version been released with the UserDataHandler change?

**Comment by elharo** [comment]
No. Code review comments have not been responded to. This PR is not acceptable as is.

**Comment by elharo** [comment]
This was addressed by another contributor.

**Pull request closed without merging**